packages/backend/src/api/v1/tabs.controller.spec.ts

```
@@ -0,0 +1,103 @@
+
+  beforeEach(() => {
+    root = createSpreadsheetDocument(); // { tab-1: "Sheet1" }
+    // Fake Yorkie doc: getRoot/update both operate on the in-memory root, so
+    // the controller mutations run exactly as they would inside doc.update.
+    const doc = {
+      getRoot: () => root,
+      update: (fn: (r: SpreadsheetDocument) => void) => fn(root),
+    };
+    const yorkieService = {
+      withDocument: jest.fn(async (_id: string, cb: (d: typeof doc) => unknown) =>
+        cb(doc),
```

ESLint

[error] Replace `async·(_id:·string,·cb:·(d:·typeof·doc)·=>·unknown)·=>⏎·······` with `⏎········async·(_id:·string,·cb:·(d:·typeof·doc)·=>·unknown)·=>`

(prettier/prettier)